edx_sga/static/css/edx_sga.css

```diff
@@ -65,15 +69,17 @@
 }
 
 .sga-block td .upload button {
-    padding: 3px;
+    padding: 7px;
     width: 100%;
 }
 
-.sga-block .error,
-#staff-graded-assignment-edit .error {
+.sga-block .error {
+    display: block;
     color: red;
 }
 
+.sga-block .error
```

Comment:
I think you accidentally left this

Reply:
Ah, I forgot to turn on my CSS logs again!